recipes/kmsxx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "kmsxx" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomba/kmsxx.git
+  git_rev: master
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  run_exports:
+    - {{ pin_subpackage('kmsxx', max_pin='x') }}
```

Comment:
This basically means that the ABI changes only for major releases, are you sure about that? Tipically to play safe if there is no info on ABI stability is available, a max_pin of x.x.x is used.